code/modules/antagonists/superheroes/buzzon/buzzon_spells.dm

```diff
@@ -0,0 +1,37 @@
+/obj/item/implant/spell/specified_type
+	var/spell_type = /obj/effect/proc_holder/spell/aoe_turf/conjure/creature/bee/buzzon
+
+/obj/item/implant/spell/specified_type/implant(mob/living/target, mob/user, silent = FALSE, force = FALSE)
+	spell = new spell_type()
+	. = ..()
+
+
+/obj/item/implant/spell/specified_type/bees
+	name = "bluespace bee storage implant"
+	desc = "Allows user to create swarms of bees on will."
+
+/obj/item/implant/spell/specified_type/bees/get_data()
+	var/dat = {"<b>Implant Specifications:</b><BR>
+				<b>Name:</b> Bluespace Bee Storage Implant<BR>
```

Comment:
why is it a STORAGE implant if the bees are being CREATED?

Reply:
Its a bluespace bee hive shoved into an implant.

Reply:
at least that was the joke